webapp/src/services/api/studies/raw/index.ts

```diff
@@ -0,0 +1,14 @@
+import { RAW_API_URL } from "../../constants";
+import { format as formatText } from "../../../../utils/stringUtils";
+import client from "../../client";
+import type { RawDownloadParams } from "./types";
+
+export async function downloadRaw(params: RawDownloadParams) {
+  const { studyId, ...rest } = params;
+  const url = `${formatText(RAW_API_URL, { studyId })}/download`;
+  const res = await client.get<Blob>(url, {
+    params: rest,
+    responseType: "blob",
+  });
+  return res.data;
+}
```

Comment:
at first look it's difficult to read and understand, please simplify to:

```typescript

export async function downloadFile(
  studyId: StudyMetadata["id"],
  path: string,
  format?: "tsv" | "xlsx",
  header?: boolean,
  index?: boolean,
) {
  const params = { path, format, header, index };

  const res = await client.get<Blob>(`v1/studies/${studyId}/raw/download`, {
    params,
    responseType: "blob",
  });

  return res.data;
}
```
rename `downloadFile` and move it the utils file of the `DownloadButton` component